test/integration_test.py

```diff
@@ -1562,15 +1563,39 @@ def assert_manifest(expected_bundles):
                 all_found_bundles.update(found_bundles)
             self.assertEqual(expected_bundles, all_found_bundles)
 
+        all_bundles = {public_bundle, *managed_access_bundles}
         # With authorized credentials, all bundles included in the filters
         # should be represented in the manifest
         with self._service_account_credentials:
-            assert_manifest({public_bundle, *managed_access_bundles})
+            test_compact_manifest(all_bundles)
 
         # Without credentials, only the public bundle should be represented
-        assert_manifest({public_bundle})
+        test_compact_manifest({public_bundle})
 
-        if ManifestFormat.curl in self.metadata_plugin(catalog).manifest_formats:
+        def dump_verbatim_manifest(bundles: Iterable[str]) -> set[str]:
+            manifest_url = furl(url=endpoint, path='/manifest/files', args={
+                'catalog': catalog,
+                'format': ManifestFormat.verbatim_jsonl.value,
+                'filters': json.dumps({special_fields.bundle_uuid: {'is': list(bundles)}})
+            })
+            manifest_lines = BytesIO(self._get_url_content(PUT, manifest_url)).readlines()
+            manifest_content = {
+                json.dumps(json.loads(replica), sort_keys=True)
+                for replica in manifest_lines
+            }
+            self.assertEqual(len(manifest_lines), len(manifest_content))
+            return manifest_content
+
+        for manifest_format in ManifestFormat.verbatim_jsonl, ManifestFormat.verbatim_pfb:
```

Comment:
L, perhaps just `format` to shorten the line, with the small context I think it's still evident that is of the manifest.
```suggestion
        for manifest_format in ManifestFormat.verbatim_jsonl, ManifestFormat.verbatim_pfb:
```